src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -142,6 +145,19 @@ private static boolean containsDefaultCaseLabelElement(DetailAST detailAst) {
         }).isPresent();
     }
 
+    /**
+     * Checks if a switch block contains a null case label.
+     *
+     * @param detailAst first case group to check.
+     * @return true if switch block contains null case label
+     */
+    private static boolean containsNullCaseLabelElement(DetailAST detailAst) {
+        return TokenUtil.findFirstTokenByPredicate(detailAst, ast -> {
+            return ast.getFirstChild() != null
+                     && verifyNullCaseLabel(ast.getFirstChild()) != null;
```

Comment:
Shouldn’t this be the actual token, LITERAL_NULL?

Reply:
@nrmancuso I have already tried using this way, but this is not working 

```
 private static boolean containsNullCaseLabelElement(DetailAST detailAst) {
        return TokenUtil.findFirstTokenByPredicate(detailAst, ast -> {
            return ast.getFirstChild() != null
                     && ast.getFirstChild().findFirstToken(TokenTypes.LITERAL_NULL) != null;
        }).isPresent();
    }
```

Reply:
>but this is not working

You will need to examine the AST and see why, it is likely that LITERAL_NULL is a child of EXPR, as opposed to pattern variable definitions.